### ISSUE TITLE
(Parent #217) Kill a continuous feed Changes client

### ISSUE THREAD

**Issue by bitsandbytes** [mobdata/replication#219]
The lightcouch Changes API supports CouchDb's continuous feed, where a client can listen "forever" for the next change. This is ideal for our test cases since we want to keep listening and report anything that comes across. 

A loop is created, processing each change then listening/waiting for another any change.

As long as the client keeps looping after each change we have an opportunity to check if the test is over and it's ready to exit the loop.

But there's no guarantee a change comes in. If, for example, we are testing to ensure a replication rule prevents documents from getting out, then "no change" is good news.

We need to then kill this blocking thread so the test can exist.

What to do:
* Add a method to MdChanges called `getContinuousFeedChanges()`
* Using the "feed type continuous" example here, http://www.lightcouch.org/lightcouch-guide.html#changes, configure as such:
  * `.includeDocs(false).heartBeat(30000)`, note we are not using a filter here
  * Upon getting a change use the "stats" log to log the doc ID and REV.
* Create a test
  * In the test, create an MdChanges using whatever dbClient.
  * Call MdChanges.getContinuousFeedChanges()
  * Wait by sleeping for like 5 seconds
When running the test, manually verify the "stats" log never logs anything from continuous feed. Since no documents are saved nothing should ever be received/stored.
* This task doesn't produce a production-ready solution, so it shouldn't be merged into the master branch.
